config/locales/en.yml

```
@@ -521,15 +521,9 @@ en:
     in_progress: in progress
     not_started: not started
   type_of_answer:
-    routing_warning_about_change_answer_type_html: |
-      <p class="govuk-body">
-        Changing the type of answer you need from “Selection from a list of options” to a different answer type means your
-        question route will no longer work and will be deleted.
-      </p>
-      <p class="govuk-body">
-        If you do not want to lose your question route you can cancel this change by using the back button or
-        clicking the “Go to your questions” link.
-      </p>
+    routing_warning_about_change_answer_type_heading: Your existing question route may be deleted
+    routing_warning_about_change_answer_type_html: "<p class=\"govuk-body\">\n  Changing the answer type from \"Selection from a list of options\" to a different answer \n  type means your route will no longer work and will be deleted.\n</p>\n<p class=\"govuk-body\">\n  If you do not want to lose your question route you can cancel this change by using the back button \n  or clicking the \"Go to your questions\" link.\n</p>\n"
```

Might be good to keep the whitespace in here for readability:

```suggestion
    routing_warning_about_change_answer_type_html: |
      <p class="govuk-body">
          Changing the type of answer you need from “Selection from a list of options” to a different answer type means your route will no longer work and will be deleted.
      </p>
      <p class="govuk-body">
        If you do not want to lose your question route you can cancel this change by using the back button or clicking the “Go to your questions” link.
      </p>
```

---

Resolving as Al informs me this is `i18n-tasks normalise` output